website/docs/docs/dbt-versions/release-notes/03-Oct-2023/custom-branch-fix-rn.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Fix: Default behavior for CI job runs without a custom branch"
+description: "October 2023: CI job runs now default to the main branch of the Git repository when a custom branch isn't set"
+sidebar_label: "Fix: Default behavior for CI job runs without a custom branch"
+tags: [Oct-2023, CI]
+date: 2023-10-06
+sidebar_position: 08
+---
+
+If you don't set a [custom branch](/docs/dbt-cloud-environments#custom-branch-behavior) for your dbt Cloud environment, it now defaults to the default branch of your Git repository (for example, `main`). Previously, [CI jobs](/docs/deploy/ci-jobs) would run for pull requests (PRs) that were opened against _any branch_ or updated with new commits if the **Custom Branch** option wasn't set. 
```

Comment:
Is there any potential behavior a user could see that we need to call out (for example, potential information might go missing since it's now pulling from a default branch, rather than all of them)?

```suggestion
If you don't set a [custom branch](/docs/dbt-cloud-environments#custom-branch-behavior) for your dbt Cloud environment, it now defaults to the default branch of your Git repository (for example, `main`). Previously, [CI jobs](/docs/deploy/ci-jobs) would run for pull requests (PRs) that were opened against _any branch_ or updated with new commits if the **Custom Branch** option wasn't set. 
```

Reply:
checked with product team. no information would be missing